app/src/types/api.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2025 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export interface ApiError {
+  errorCode: number;
+  message: string;
+}
+
+export interface ProjectsSearchesItem {
+  id: number;
+  key: string;
+  organization_id: number;
+  created_at: string;
+  updated_at: string;
+  stats: unknown;
+  name: string;
+  slug: string;
+}
+
+export interface ProjectsSearchesResponseData {
+  offset: number;
+  limit: number;
+  total_count: number;
+  sort: string;
+  order: string;
+  items: ProjectsSearchesItem[];
+}
```

Comment:
Does it make sense to store the specific api endpoint related types in the corresponding `controller`s directory?